src/main/java/com/greenfox/dramacsoport/petclinicbackend/config/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,55 @@
+package com.greenfox.dramacsoport.petclinicbackend.config;
+
+import com.greenfox.dramacsoport.petclinicbackend.dtos.ErrorResponse;
+import com.greenfox.dramacsoport.petclinicbackend.exceptions.DeletionErrorException;
+import com.greenfox.dramacsoport.petclinicbackend.exceptions.PasswordException;
+import com.greenfox.dramacsoport.petclinicbackend.exceptions.UnauthorizedActionException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import javax.naming.NameAlreadyBoundException;
+
+@ControllerAdvice
+public class GlobalExceptionHandler {
+
+    @ExceptionHandler(DeletionErrorException.class)
+    public ResponseEntity<ErrorResponse> handleDeletionError(DeletionErrorException ex) {
```

Comment:
I do have some questions about the `DeletionErrorException` classname. Shouldn't it be exception or error, but not both?!